torch/quantization/fx/qconfig_utils.py

```diff
@@ -134,6 +165,14 @@ def generate_qconfig_map(
             module_path, module_type = node_name_to_scope[node.name]
             qconfig = get_qconfig(
                 qconfig_dict, module_type, module_path, function_qconfig)
+
```

Comment:
I think the precedence should probably be:
function_order_config > global_function_type_config > module_scope_config > global_config

Reply:
where module_scope_config is from module_name_config or global module_type_config and the precedence there is module_name_config > module_type_config (this is taken care of by get_qconfig function I think)

Reply:
> I think the precedence should probably be: function_order_config > global_function_type_config > module_scope_config > global_config

yes, this is true, although not the easiest to see with the current code.  Saving cleaning up the qconfig tools to make the logic easier to understand for a future PR, hope that's ok!